vmware-event-router/README.MD

```diff
@@ -18,11 +18,22 @@ The VMware Event Router uses the [CloudEvents](https://cloudevents.io/) standard
 to normalize events from the supported event `providers`. See
 [below](#example-event-structure) for an example.
 
+**Event Delivery Guarantees:**
+- At-least-once event delivery for the [vCenter event provider](#provider-type-vcenter) (using checkpoints)
+- At-most-once event delivery ("stateless" mode)
+
 **Current limitations:**
 
 - Only one event `provider` and one event `processor` can be configured at a
   time
-- **At-most-once** event delivery semantics
+- At-least-once event delivery semantics cannot be guaranteed if the event
+  router crashes **within seconds** right after having received some events but before creating the
+  first valid checkpoint
+- At-least-once event delivery semantics are only supported for the vCenter
+  event provider
+- At-least-once event delivery semantics currently only guard against crashes of
```

Comment:
This really depends on how you deploy the router, e.g. in a container, native binary, etc. So I would say these details are out of scope for this PR because it depends on the end user's environment. I spoke with @rguske about creating a blog post on how to deploy the router on Kubernetes with a persistent volume, which we can reference and share then.